snapcraft/internal/repo/_platform.py

```diff
@@ -0,0 +1,36 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import platform
+
+from ._deb import Ubuntu
+
+_DEB_BASED_PLATFORM = [
+    'Ubuntu',
+    'Debian',
+]
+
+
+def _is_deb_based():
+    return platform.linux_distribution()[0] in _DEB_BASED_PLATFORM
```

Comment:
thanks, this will be a problem wrt wanting to have single packaging across xenial, yakkety and zesty but I'll figure it out

```
$ rmadison python3-distro
 python3-distro | 1.0.1-2 | zesty/universe | all
```